src/sentry/api/endpoints/organization_code_mappings.py

```diff
@@ -133,15 +133,18 @@ def get(self, request: Request, organization) -> Response:
 
         integration_id = request.GET.get("integrationId")
         project_id = request.GET.get("projectId")
+        all_items = request.GET.get("allItems")
```

Comment:
does anyone have more descriptive suggestions for `allItems`?

Reply:
Can you elaborate on what `allItems` is? All the repo project path configs?

Reply:
@ceorourke yea all the code mappings for the organization. But its probably safer to fetch for all code mappings that the user has access to.

Reply:
Hmmm then maybe `allCodeMappings` or `userScopedCodeMappings`?

Reply:
@ceorourke I ended up just using our current standard of `projectId=-1` to denote `find for all projects the user has access to`